polyfills/HTMLCanvasElement/protoype/toBlob/config.json

```diff
@@ -0,0 +1,11 @@
+{
+	"browsers": {
+		"chrome": "<50",
+		"firefox": "<19",
+		"safari": "*"
+	},
+	"dependencies": [
+		"Object.defineProperty"
```

Comment:
The polyfill also makes use of `atob`, which we serve a polyfill for. Could you add `"atob"` to the list of dependencies please?